tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1235,6 +1270,18 @@ When(
   }
 )
 
+When('the user selects following files/folders to copy/move using the webUI', async function(
+  resources
+) {
+  for (const item of resources.rows()) {
+    await client.page.FilesPageElement.filesList().toggleFileOrFolderCheckbox('enable', item[0])
+  }
+})
+
+When('the user uses the copy button to copy files using the webUI', function() {
```

Comment:
I think something like `the user clicks the copy button...` sounds better here.

Reply:
But we don't use click normally, do we?